### PR TITLE
Add icons to integrations markdown files

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

This PR adds the icon of every integration, as pulled from `https://netdata.cloud/img`.
I selected a width for all icons of 150 pixels

The result in Learn will look like this :point_down (125% zoomed screenshot): 

image

### CODE REVIEW

**Comment by Ancairon** [comment]
tested at https://github.com/netdata/netdata/pull/16170

File: integrations/gen_docs_integrations.py [hunk 1]
```diff
@@ -117,6 +117,25 @@ def read_integrations_js(path_to_file):
         print("Exception", e)
 
 
+def create_img_link(integration, filename):
```
Comment (ilyam8) [inline integrations/gen_docs_integrations.py:120]: why passing filename? it is always `integration['meta']['monitored_instance']['icon_filename']` and you pass integration!

Comment (Ancairon) [reply integrations/gen_docs_integrations.py:120]: no it ain't, exporters and notifications don't have monitored instance

File: integrations/gen_docs_integrations.py [hunk 2]
```diff
@@ -168,6 +188,12 @@ def build_readme_from_integration(integration, mode=''):
             meta_yaml = integration['edit_link'].replace("blob", "edit")
             sidebar_label = integration['meta']['name']
             learn_rel_path = generate_category_from_name(integration['meta']['categories'][0].split("."), categories)
+
+            split = re.split(r'(#.*\n)', integration['overview'], 1)
+
+            first_overview_part = split[1]
+            rest_overview_part = split[2]
```
Comment (ilyam8) [inline integrations/gen_docs_integrations.py:195]: This is not needed

File: integrations/gen_docs_integrations.py [hunk 3]
```diff
@@ -200,6 +225,12 @@ def build_readme_from_integration(integration, mode=''):
...
```
Comment (ilyam8) [inline integrations/gen_docs_integrations.py:232]: not needed

File: integrations/gen_docs_integrations.py (hunk 2 above)
Comment (Ancairon) [reply integrations/gen_docs_integrations.py:195]: true

File: integrations/gen_docs_integrations.py (hunk 3 above)
Comment (Ancairon) [reply integrations/gen_docs_integrations.py:232]: fixing!

**Review by ilyam8 — APPROVED** [review]

**Comment by tkatsoulas** [comment]
What about distortion from "scale" to 150, to you think that we may have any problem with icons?

**Review by sashwathn — APPROVED** [review]

**Pull request merged**